packages/ui/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
This triggers an ESLint warning since we don't have synthetic default exports turned on and React doesn't have a default export.

Instead, can we just import `FC`?
```suggestion
import { FC } from 'react'
```